### PR TITLE
linchpin: improve controller provisioning example

### PR DESCRIPTION
**Pull request by rvykydal**
Depends on https://github.com/rhinstaller/kickstart-tests/pull/292

### CODE REVIEW

File: linchpin/examples/controller-provisioning-linchpin/README.md [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
-Also set credentials to be passed to the *controller* in `<PUBLIC_KEY_TO_UPLOAD_PATH>`, `<PRIVATE_KEY_TO_UPLOAD_PATH>`, `<CLOUD_CONFIG_FILE>` (or comment out the lines if you don't intend to use them on the controller).
+- Update the deployment key name in [PinFile.controller](PinFile.controller) variable `keypair` to the keypair you want to use.
 
-Run [kstests-controller-provision.sh](kstests-controller-provision.sh) in `kickstart-tests` repo root.
+- Copy [kstests-controller-provision.sh](kstests-controller-provision.sh) in `kickstart-tests` repo root and configure the inventory:
+    - Replace the `<PRIVATE_KEY_PATH>` with path to the private key of the cloud keypair used above (if it is not your default key).
+    - Replace the `<PUBLIC_KEY_PATH>` with ssh public key for controller user (`kstest`) access.
+    - Replace the `<CLOUD_CONFIG_FILE>` with path to cloud configuration file and uncomment the variable or place the file to the default location as configured in [kstest-controller/defaults/main.yml](../../../ansible/roles/kstest-controller/defaults/main.yml).
```
Comment (jkonecny12) [inline linchpin/examples/controller-provisioning-linchpin/README.md]: I think it should be ``..path to *the* cloud configuration file...``

File: linchpin/examples/controller-provisioning/README.md [hunk 2]
```diff
@@ -1,34 +1,38 @@
...
+It may be useful to pass also an ssh keypair to be further used to the controller. The keys to be passed to the controller user can be configured by the playbook variables `private_keys_to_upload`, `public_keys_to_upload` that can be set for example in the [inventory](hosts).
 
-## Provision and deploy:
+### Provisioning and deployment
 
-Modify ssh key configuration in the [ks.kstest-controller.cfg](ks.kstest-controller.cfg) for your keypair.
+- Modify ssh key configuration in the [kickstart](ks.kstest-controller.cfg) for your keypair.
 
-Install the controller host using the [ks.kstest-controller.cfg](ks.kstest-controller.cfg) kickstart.
+- Install the controller host using the [kisckstart](ks.kstest-controller.cfg).
```
Comment (jkonecny12) [inline linchpin/examples/controller-provisioning/README.md]: `kisckstart` -> `kickstart`

File: linchpin/examples/controller-provisioning-linchpin/README.md (hunk 1 above)
Comment (rvykydal) [reply linchpin/examples/controller-provisioning-linchpin/README.md]: Fixed

File: linchpin/examples/controller-provisioning/README.md (hunk 2 above)
Comment (rvykydal) [reply linchpin/examples/controller-provisioning/README.md]: Fixed

**Pull request merged**